### ISSUE TITLE
this plugin conflicts with vim built-in search behavior

### ISSUE THREAD

**Issue by pujfei** [henrik/vim-indexed-search#25]
type / in vim normal mode and search a string, with vim's genuine ability, you can go to the n-th search by pressing 'n' or 'N' after a given number.

while with this plugin turned on, the genuine key binding does not work, giving warning: E481: No range allowed.

can this be improved to make it compatitable?

**Comment by henrik** [comment]
Hm, this works for me™.

If I do "/foo" it says e.g. "First of 8 matches". If I then hit "n" it says "Match 2 of 8". If I then hit "2n" it says "Match 4 of 8". If I hit "2N" it says "Match 2 of 8".

I'm at revision 5af020bba084b699d0453f242d7d76711d64b1e3, which is latest master. Vim 8.2.

**Comment by pujfei** [comment]
Good day @henrik ,

What is the linux release you are using? I am working on a WSL Ubuntu 20.04 (Windows Subsystem for Linux).

Besides, it seems that from Vim 8.1.1270 on, VIM has recently integrated this "vim-indexd-search" plugin functionality into its inherent ability, just by adding this line in the .vimrc file:

`set shortmess-=S`

https://stackoverflow.com/questions/4668623/show-count-of-matches-in-vim

This topic was posted in 2011, which is ten years ago, while the vimrc option was added around Feb 13 this year.

I have tried adding that line to vimrc, it works nice and smoothly.
